core/src/actions/records.ts

```diff
@@ -143,6 +143,26 @@ export class RecordCreate extends AuthenticatedAction {
     if (params.properties) {
       await record.addOrUpdateProperties(params.properties);
     }
+
+    await record.markPending();
+    await record.import();
+    await record.updateGroupMembership();
+    await record.update({ state: "ready" });
```

Comment:
Is this a use case for `RecordOps.importAndUpdateInline`?

Reply:
I gave it a shot, but the behavior is a little different:
1. we can't use a transaction, which we want to do
2. we do want to throw if something goes wrong with the import (`record.import()` throws).  `importAndUpdateInline` does not throw and tries to keep going

Reply:
I did a little rename of the method to make that more clear -> `importAndUpdateInline -> opportunisticallyImportAndUpdateInline`